### PR TITLE
Fix testgrid web

### PR DESCRIPTION
**Pull request by emosbaugh**
#### What type of PR is this?

type::bug

#### What this PR does / why we need it:

Fixes testgrid web upgrading react and react router

### CODE REVIEW

File: testgrid/web/src/components/views/InstanceTable.jsx [hunk 1]
```diff
@@ -14,29 +15,60 @@ import Loader from "../shared/Loader";
...
+  const [showLogsModal, setShowLogsModal] = useState(false);
+  const [loadingLogs, setLoadingLogs] = useState(false);
+  const [showSonobuoyResultsModal, setShowSonobuoyResultsModal] = useState(false);
+  const [loadingSonobuoyResults, setLoadingSonobuoyResults] = useState(false);
+  const [showUpgradeYaml, setShowUpgradeYaml] = useState(false);
+  const [showSupportbundleYaml, setShowSupportbundleYaml] = useState(false);
+  const [showPostInstallScript, setShowPostInstallScript] = useState(false);
+  const [showPostUpgradeScript, setShowPostUpgradeScript] = useState(false);
+
+  // TODO: this does not work
```
Comment (emosbaugh) [inline testgrid/web/src/components/views/InstanceTable.jsx:39]: this still does not work but i'd like to merge this as it makes things usable again

**Review by laverya — APPROVED** [review]

**Review by jodillion — DISMISSED** [review]
I don't know what this functionality really does but the react looks good to me

File: testgrid/web/src/components/views/InstanceTable.jsx [hunk 2]
```diff
@@ -36,20 +36,20 @@ const InstanceTable = (props) => {
   const [showPostInstallScript, setShowPostInstallScript] = useState(false);
   const [showPostUpgradeScript, setShowPostUpgradeScript] = useState(false);
 
-  // TODO: this does not work
+  let logsAceEditor = null;
+
```
Comment (jodillion) [inline testgrid/web/src/components/views/InstanceTable.jsx:40]: future note to refactor this to not set the input directly in the AceEditor but for now 🤷‍♀️

**Review by laverya — APPROVED** [review]

**Review by jodillion — APPROVED** [review]
looks good, nice update

**Pull request merged**